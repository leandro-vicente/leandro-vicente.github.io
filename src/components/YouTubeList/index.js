import React from 'react'

import { SUBSCRIBE_FORM_URL } from '../../constants'

const redirectToSubscribeForm = () => {
  return (
    window.location.replace(SUBSCRIBE_FORM_URL)
  )
} 

const YouTubeList = () => {
  return(
    <React.Fragment>
      <h1>Loading...</h1>
      { redirectToSubscribeForm() }
    </React.Fragment>
  )
}

export default YouTubeList