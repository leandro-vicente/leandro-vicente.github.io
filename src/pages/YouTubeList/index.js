const YouTubeList = () => window.location.replace("https://goo.gl/forms/A3zRVgzeH3BMD2FJ3")

export default YouTubeList