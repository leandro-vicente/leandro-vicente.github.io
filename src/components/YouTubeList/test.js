import React from 'react'
import { shallow } from 'enzyme'
import YouTubeList from '.'

describe('YouTube List', () => {
  it('Renders YouTube List component correctly', () => {
    const youtubeList = shallow(<YouTubeList />)
    expect(youtubeList).toMatchSnapshot()
  })
})