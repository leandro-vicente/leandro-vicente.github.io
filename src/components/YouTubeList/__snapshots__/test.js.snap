// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`YouTube List Renders YouTube List component correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <YouTubeList />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <h1>
          Loading...
        </h1>,
        undefined,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Loading...",
        },
        "ref": null,
        "rendered": "Loading...",
        "type": "h1",
      },
      undefined,
    ],
    "type": Symbol(react.fragment),
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <h1>
            Loading...
          </h1>,
          undefined,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Loading...",
          },
          "ref": null,
          "rendered": "Loading...",
          "type": "h1",
        },
        undefined,
      ],
      "type": Symbol(react.fragment),
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
